tools/docker/Dockerfile

```diff
@@ -8,21 +8,27 @@ RUN cd ui && npm install && npm run build
 # Build app
 FROM registry.access.redhat.com/ubi8/python-39
 ARG USER_ID=${USER_ID:-1001}
-ARG DEVEL_AE_LIBRARY=0
+ARG DEVEL_AE_LIBRARY=1
 WORKDIR $HOME
 
 USER 0
+RUN yum install -y yum-utils
+RUN yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+RUN yum install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
```

Comment:
If this needed? As far I can see, aiodocker uses the REST API, doesn't require the daemon. just the socket.